src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -481,22 +479,18 @@ public MediaPatchResponse patchMediaRequest(Integer mediaRequestId, MediaPatchRe
     }
 
     @Override
-    public ResponseEntity<List<AdminMediaSearchResponseItem>> adminMediaSearch(Integer transformedMediaId, Integer transcriptionDocumentId) {
-        if (transformedMediaId != null) {
-            Optional<TransformedMediaEntity> transformedMediaOpt = transformedMediaRepository.findById(transformedMediaId);
-            if (transformedMediaOpt.isEmpty()) {
-                return new ResponseEntity<>(new ArrayList<>(), HttpStatus.OK);
-            }
-            TransformedMediaEntity transformedMedia = transformedMediaOpt.get();
-            MediaRequestEntity mediaRequest = transformedMedia.getMediaRequest();
-            HearingEntity hearing = mediaRequest.getHearing();
-            List<MediaEntity> mediaList = mediaRepository.findAllByHearingId(hearing.getId());
-            //filter by media that overlap with request
-            List<MediaEntity> filteredMediaList = mediaList.stream().filter(mediaEntity -> mediaEntity.getStart().isBefore(mediaRequest.getEndTime())
-                && mediaEntity.getEnd().isAfter(mediaRequest.getStartTime())).toList();
-            return new ResponseEntity<>(AdminMediaSearchResponseMapper.createResponseItemList(filteredMediaList, hearing), HttpStatus.OK);
-        } else {
-            return new ResponseEntity<>(HttpStatus.NOT_IMPLEMENTED);//future functionality
+    public List<AdminMediaSearchResponseItem> adminMediaSearch(Integer transformedMediaId) {
+        Optional<TransformedMediaEntity> transformedMediaOpt = transformedMediaRepository.findById(transformedMediaId);
+        if (transformedMediaOpt.isEmpty()) {
+            return new ArrayList<>();
         }
+        TransformedMediaEntity transformedMedia = transformedMediaOpt.get();
+        MediaRequestEntity mediaRequest = transformedMedia.getMediaRequest();
+        HearingEntity hearing = mediaRequest.getHearing();
+        List<MediaEntity> mediaList = mediaRepository.findAllByHearingId(hearing.getId());
+        //filter by media that overlap with request
+        List<MediaEntity> filteredMediaList = mediaList.stream().filter(mediaEntity -> mediaEntity.getStart().isBefore(mediaRequest.getEndTime())
```

Comment:
What if the start and end times match exactly? Should we include the media in such cases